framework/src/main/scala/org/sunbird/obsrv/core/util/SystemConfigSelector.scala

```diff
@@ -0,0 +1,56 @@
+package org.sunbird.obsrv.core.util
+
+import com.typesafe.config.{Config, ConfigFactory}
+import org.slf4j.LoggerFactory
+import org.sunbird.obsrv.core.model.Models.SystemSettings
+
+import java.io.File
+import java.sql.ResultSet
+
+object SystemConfigSelector {
```

Comment:
1. Rename the object as SystemSettingsService. What does selector mean?
2. Create two methods - getAllSystemSettings(), getSystemSetting(key)